examples/emg_and_marker_tracking/emg_tracker.py

```diff
@@ -115,7 +115,7 @@ def prepare_ocp(
         raise RuntimeError("Wrong choice of kin_data_to_track")
 
     # Dynamics
-    variable_type = ProblemType.muscles_and_torque_driven
+    variable_type = ProblemType.excitation_driven
```

Comment:
Unless there is no residual torque in your dynamic function, this name is misleading (so is the previous one). Could you change that for `muscle_excitations_and_torque_driven` and modify the previous one for `muscle_activations_and_torque_driven`. Make sure you change it everywhere needed